### ISSUE TITLE
Can't compile in Xcode 10

### ISSUE THREAD

**Issue by zhouhao27** [chriseidhof/commonmark-swift#26]
Got the following error message when compiling:

> /Users/Projects/test/MarkDown/MarkDown/ViewController.swift:10:8: No such module 'CommonMark'

Here is my steps to add CommonMark:

1. Create a single view project 
2. `pod init`
3. Add `pod 'CommonMark'` in my `Podfile`
4. `pod install`
5. Open workspace
6. Add `import CommonMark` in my `ViewController.swift`

Then got this error message. Any suggestion? Thanks.

**Comment by chriseidhof** [comment]
Hm, we currently don't support CocoaPods, you can only install through SPM.

**Comment by zhouhao27** [comment]
Thanks for your reply. Not familiar with how to use SPM. Any guideline? 

I can't compile CommonMark in Xcode project. It shows:

> /Users/Projects/test/MarkDown/commonmark-swift/Sources/Ccmark/module.modulemap:2:12: Header '/usr/local/include/cmark.h' not found

Need to install Cmark first?

**Comment by chriseidhof** [comment]
Yes, you need cmark installed. So currently it won't work on iOS unless you manually add those files as well.
